packages/web-components/fast-components/src/radio/fixtures/base.html

```diff
@@ -0,0 +1,47 @@
+<fast-design-system-provider use-defaults>
+    <h1>Radio</h1>
+
+    <h4>Defaults</h4>
+    <fast-radio></fast-radio>
+    <div>
+        <fast-radio>label</fast-radio>
+    </div>
+
+    <h4>Checked</h4>
+    <fast-radio value="checked" checked></fast-radio>
+
+    <!-- Required -->
+    <h4>Required</h4>
+    <fast-radio value="required" required></fast-radio>
+
+    <!-- Disabled -->
+    <h4>Disabled</h1>
+    <fast-radio disabled></fast-radio>
+    <fast-radio disabled>label</fast-radio>
+    <fast-radio disabled checked>checked</fast-radio>
+
+    <h4>Inline</h4>
+    <fast-radio name="fruit-inline" value="apples" checked>Apples</fast-radio>
+    <fast-radio name="fruit-inline" disabled value="bananas">Bananas</fast-radio>
+    <fast-radio name="fruit-inline" value="honeydew">Honeydew</fast-radio>
+
+    <h4>Vertical</h4>
+    <fieldset style="display: flex; flex-direction: column; align-items: start;">
```

Comment:
I wonder if now that we're adding a `radiogroup` component these examples may be more beneficial there? We can likely limit this file to single instances of radio and the variations.

Reply:
I'll trim this one up before I push the next commit